.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
```

Comment:
not sure if this is supported on vscode yet, might be in the pipeline, probably would be useful to mimic those settings in vscode configuration for now